celery/backends/base.py

```diff
@@ -273,7 +274,9 @@ def exception_to_python(self, exc):
                     exc_module = from_utf8(exc_module)
                     exc_type = from_utf8(exc['exc_type'])
                     try:
-                        cls = getattr(sys.modules[exc_module], exc_type)
+                        cls = sys.modules[exc_module]
```

Comment:
Should we leave a comment here? Maybe we can extract this to a helper to clarify the code's intentions instead?
What do you think?

Reply:
I've added comment, not sure if helper is useful here as it seems quite specific case...

Reply:
Yes, that should do it.